.storybook/preview.js

```diff
@@ -23,7 +24,10 @@ export const parameters = {
     },
   },
   viewport: {
-    viewports,
+    viewports: {
+      ...customViewports,
+      ...INITIAL_VIEWPORTS,
+    },
```

Comment:
this adds some extra viewports into storybook to make development easier for specific devices, and the option to reset the viewport